Formula/Pelikan.rb

```diff
@@ -0,0 +1,22 @@
+class Pelikan < Formula
+  desc "Pelikan: production-ready cache services"
+  homepage "http://pelikan.io"
+  url "https://github.com/twitter/pelikan/archive/0.1.1.tar.gz"
+  sha256 "98ab698b80e6e7634245e82798a9672b4a0b9f7f603879b398a66a4d97550e7f"
+  head "https://github.com/twitter/pelikan.git"
+
+  depends_on "cmake" => :build
+  depends_on "check" => :build
+
+  def install
+    mkdir "_build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "-j"
```

Comment:
Please remove the `-j` argument. Homebrew automatically passes it via `MAKEFLAGS`.